AdobeUi/view/adminhtml/web/js/components/grid/column/overlay.js

```diff
@@ -3,11 +3,44 @@
  * See COPYING.txt for license details.
  */
 define([
-    'Magento_Ui/js/grid/columns/column'
-], function (Column) {
+    'Magento_Ui/js/grid/columns/column',
+    'ko'
+], function (Column, ko) {
     'use strict';
 
     return Column.extend({
+
+        /**
+         * Returns container styles to given record.
+         *
+         * @param {Object} record - Data to be preprocessed.
+         * @returns {Object}
+         */
+        getStyles: function (record) {
```

Comment:
@konarshankar07 is there a way to apply styles using CSS from `AdobeStockAdminImageUi` module?

Reply:
@sivaschenko , Its quite difficult with the current DOM structure because the images are not in the same height and width so common CSS will not work but I will try and update you if I find any way to get this done via CSS.

Reply:
Thank you!